merge develop into dash

quinn-dougherty opened the pull request:
catch up

lottspot committed:
Workflows to deploy tagged releases to preprod and prod

bixu committed:
WIP: support running CHIME under Habitat

bixu committed:
WIP: add minimal documentation

lottspot committed:
Version bumps in k8s/app.yaml are not environment specific

lottspot committed:
Minimize difference between prod and preprod manifest trees

lottspot committed:
Remove duplicated preprod tree

lottspot committed:
Fix version reference in git commit job

quinn-dougherty committed:
first push of issue 255

quinn-dougherty committed:
it now asks user date of first hospitalization

quinn-dougherty committed:
Merge branch 'develop' into quinn_issue_255

quinn-dougherty committed:
incremental progress

quinn-dougherty committed:
Merge branch 'quinn_issue_255' of https://github.com/CodeForPhilly/chime into quinn_issue_255

quinn-dougherty committed:
code is in, have to reset x axis

quinn-dougherty committed:
day zero of graph is now the user inputted date_first_hospitalized

quinn-dougherty committed:
fixed doubling time inference

quinn-dougherty committed:
Merge branch 'develop' into quinn_issue_255

quinn-dougherty committed:
fixed graph

quinn-dougherty committed:
fixed download csv button and improved table situation

quinn-dougherty committed:
Merge branch 'develop' into quinn_issue_255

quinn-dougherty committed:
incremental progress on debugging

quinn-dougherty committed:
incremental progress debugging dates

quinn-dougherty committed:
Merge branch 'develop' into quinn_issue_255

quinn-dougherty committed:
relative_contact_pct_input from relative_contact_rate_input

quinn-dougherty committed:
Merge branch 'quinn_issue_255' of https://github.com/CodeForPhilly/chime into quinn_issue_255

quinn-dougherty committed:
began argmin_days_since

quinn-dougherty committed:
i think i completed argmin over days_since?

jlubken committed:
Allow negative days from n_days_since to flow through dfs

jlubken committed:
Fix some tests

jlubken committed:
Replace add_date_column with numpy astype

jlubken committed:
Move input for current date

lottspot committed:
Prevent downtime when deploying new versions of app

Commit:
Fix setup.py from merge

jlubken committed:
Add date to dispositions_df and flow date through

jlubken committed:
Replace today with current_date, add recovery_days

jlubken committed:
Fix setup.py

Fix setup.py

jlubken committed:
Replace recovery_days with infectious_days

jlubken committed:
Add settings.cfg for infectious_days

jlubken committed:
Fix display_mode_info and infectious_days

themightychris committed:
Merge branch 'develop' into feature/habitat_service_support

jlubken committed:
Simplify chart descritpions

themightychris committed:
Merge pull request #262 from bixu/feature/habitat_service_support

WIP: support running CHIME under Habitat

jlubken committed:
Fix census_df bug

themightychris committed:
fix: update habitat plan for setuptools

themightychris committed:
chore: update plan origin/maintainer to Code for Philly

themightychris committed:
chore: remove sudoers workaround that seems to no longer be needed

themightychris committed:
feat: generate habitat artifact version from git tag

jlubken committed:
Fix SIR chart

jlubken committed:
Remove more presentation cruft

jlubken committed:
Fix reference to p.current_date

themightychris committed:
feat: configure streamlit via habitat and export port

jlubken committed:
Alphabetized state

jlubken committed:
Add mandatory named parameters to prevent mistakes

themightychris committed:
chore: add pytest .cache to .gitignore and remove .cache/

themightychris committed:
chore: remove duplicate .gitignore rule

themightychris committed:
chore: remove someone's local vscode workspace

themightychris committed:
chore: add portable vscode workspace configuration

themightychris committed:
chore: remove vestigial setup.sh

themightychris committed:
fix: restore primary Procfile for heroku

themightychris committed:
docs: move manual validation procedure into docs

jlubken committed:
Remove as_date

themightychris committed:
style: remove footer link syntax in docs TOC

jlubken committed:
Remove modulo sliders

themightychris committed:
fix: restore original Procfile with PORT handling

themightychris committed:
chore: gitignore cypress videos/screeshots

themightychris committed:
feat: enable `npm run test` script

themightychris committed:
refactor: externalize cypress baseUrl

jlubken committed:
Fix easy tests

themightychris committed:
Merge pull request #307 from CodeForPhilly/fixes/workspace-cleanup

Clean up workspace and restore Heroku deploy

jlubken committed:
Fix cli

jlubken committed:
Add logger

jlubken committed:
Add fstrings

jlubken committed:
Remove assertions

jlubken committed:
Move change date and type, fix annotation

jlubken committed:
Add change date to cli

jlubken committed:
Remove bug hiding spot

jlubken committed:
Add todo for susceptible bug check

jlubken committed:
Fix more tests

Commit:
Correct the fitting procedure

jlubken committed:
Fix moar tests

jlubken committed:
Correct the fitting procedure

Correct the fitting procedure

jlubken committed:
Ensure i_days, and n_days_since exist in model

jlubken committed:
Add logging for set doubling_time

jlubken committed:
Add current day ruler to census and sim_sir charts

jlubken committed:
Remove current_date ruler

jlubken committed:
Add working verical rule for current date

jlubken committed:
Move constants to reduce circular dependencies

jlubken committed:
Add current day to admits chart

themightychris committed:
docs: add Habitat to operations guides

themightychris committed:
style: delint habitat readme

themightychris committed:
style: add terminal newline

themightychris committed:
Merge pull request #325 from CodeForPhilly/features/bindable-habitat

Bindable Habitat deploy option

jlubken committed:
Fix test

jlubken committed:
Update setup for mypy, long_description

jlubken committed:
Fix project reference

quinn-dougherty committed:
changed p.current_hospitalized to 1 on line 58

quinn-dougherty committed:
fixed model_last test

jlubken committed:
Fix another test

quinn-dougherty committed:
fixed test_model test

quinn-dougherty committed:
Merge branch 'more_issue_255' of https://github.com/CodeForPhilly/chime into my_more_issue_255

jlubken committed:
Remove duplicate computation

jlubken committed:
Replace n_days_since with i_day

Commit:
Put social distancing effect back in. Starts at current day.

jlubken committed:
More logging of givens and estimates

jlubken committed:
Log len df and expected range

jlubken committed:
Resolve changes from upstream

jlubken committed:
Fix test reflecting social distancing

jlubken committed:
Add social distancing

Put social distancing effect back in. Starts at current day.

jlubken committed:
Move n_days back to 60 so social distancing can be seen in the plots

Commit:
Remove confirmed cases & detection prob

These are irrelevent to the calculations and only cause user confusion.

jlubken committed:
Update change_date notice

jlubken committed:
Copy params for tests

jlubken committed:
Add copy for "constant" PARAM to MODEL and HALVING_MODEL

jlubken committed:
Fix tests for r_t

quinn-dougherty committed:
Merge pull request #329 from CodeForPhilly/update_setup

Update setup

jlubken committed:
Remove detection prob

Remove confirmed cases & detection prob

jlubken committed:
Patch all

jlubken committed:
Add settings

jlubken committed:
Try to fix cypress

jlubken committed:
Off by one for cypress

jlubken committed:
Add lots of time

Add time as a parameter

jlubken committed:
Resolve issue 255

jlubken committed:
Fix issue 255

Fix issue 255

jlubken committed:
Replace lengths of stay with days

lottspot committed:
Show kubernetes resource diffs before merging release

jlubken committed:
Add conservation test

jlubken committed:
Remove duplicate conservation tests

jlubken committed:
Add Conservation Test

Add conservation test

jlubken committed:
Merge branch 'develop' into replace_los_with_days

jlubken committed:
Update help text

jlubken committed:
Show help

jlubken committed:
Update Average Days on Ventilator text

Co-Authored-By: ecsmyth <17688672+ecsmyth@users.noreply.github.com>

jlubken committed:
Update Average Days in ICU text

Co-Authored-By: ecsmyth <17688672+ecsmyth@users.noreply.github.com>

jlubken committed:
Update Average Hospital Length of Stay text

Co-Authored-By: ecsmyth <17688672+ecsmyth@users.noreply.github.com>

jlubken committed:
Replace los with days

Replace lengths of stay with days

jlubken committed:
Add disposition type

PhilMiller committed:
Check population monotonicity conditions

jlubken committed:
Fix drifting p.current_date

jlubken committed:
Remove print

jlubken committed:
Check population monotonicity

Check population monotonicity

jlubken committed:
Merge branch 'develop' into add_disposition_type

jlubken committed:
Add comments for the Disposition type tuple

jlubken committed:
Replace RateDays/RateLos with Disposition

Replace RateDays/RateLos with Disposition

themightychris committed:
Merge pull request #298 from CodeForPhilly/issues/297

Prevent downtime when deploying new versions of app

Commit:
presentation: black formatting

lottspot committed:
Color code prod_diff in deploy workflow

Co-Authored-By: Chris Alfano <calfano@codeforamerica.org>

lottspot committed:
Color code app_diff in deploy workflow

Co-Authored-By: Chris Alfano <calfano@codeforamerica.org>

Commit:
presentation: Updates for 2.0.0 release

Commit:
Remove Penn Medicine hospitals ref

Add note about back-casting

Commit:
Remove Penn Medicine hospitals ref

Add note about back-casting

Commit:
Add #288 to pre-dash release

lottspot committed:
Merge pull request #259 from CodeForPhilly/issues/48

Workflows to deploy tagged releases to preprod and prod

Commit:
Fix stepsize and bounds of social distancing

Commit:
Fix stepsize and bounds of social distancing

Commit:
Merge branch 'v2_prerelease_mods' of github.com:CodeForPhilly/chime into v2_prerelease_mods

Commit:
Clarify parameter label language

jlubken commented on doubling_time.cfg:
```diff
@@ -0,0 +1,14 @@
+--icu-los 9
+--icu-rate 0.0075
+--infectious-days 14
+--known-infected 510
```
Remove known_infected

jlubken commented on date_first_hospitalized.cfg:
```diff
@@ -0,0 +1,14 @@
+--icu-los 9
+--icu-rate 0.0075
+--infectious-days 14
+--known-infected 510
```
Remove known infected

Commit:
Social distancing step-size 0.25

Precision beyond this is kidding yourself.
User can still enter manually an arbitrarily precise value.

Commit:
Fix "currently infected individuals" message bug

Commit:
Update defaults

Remove CCH from default set of hospitals
This affects the regional population
Remove county-level population accounting

Commit:
Fix form validation violations at default settings

Commit:
presentation: Update release version

cjbayesian committed:
Merge pull request #370 from CodeForPhilly/fix_release_version

presentation: Update release version

lottspot committed:
k8s deployments choke on repository URLs with uppercase letters

Commit:
Update version in setup.py

MooseQuest committed:
Merge pull request #372 from CodeForPhilly/issues/371

k8s deployments choke on repository URLs with uppercase letters

jlubken committed:
Merge branch 'develop' of github.com:CodeForPhilly/chime into v2_prerelease_mods

jlubken committed:
v1.1.0-rc.2 changes

v1.1.0-rc.2 changes
